client/state/signup/dependency-store/schema.js

```diff
@@ -0,0 +1,3 @@
+export const dependencyStoreSchema = {
+	type: 'object'
```

Comment:
It's probably more useful to be more descriptive than this. For example, lets say we wanted to refactor how we store data for a particular step (perhaps siteId changes into an integer vs a string). We wouldn't know to throw out older data, if it's as generic as this. Check out [this guide](https://spacetelescope.github.io/understanding-json-schema/) to learn a bit more about json schema.

Looking at some of the data coming in, this schema might look like:
<img width="336" alt="screen shot 2016-07-25 at 11 15 47 am" src="https://cloud.githubusercontent.com/assets/1270189/17112136/95694eba-5259-11e6-9023-ee3ce0256e88.png">

```javascript
export const dependencyStoreSchema = {
	type: 'object',
	properties: {
		surveySiteType: { type: 'string' },
		surveyQuestion: { type: 'string' },
		designType: { type: 'string' },
		theme: { type: 'string' },
		cartItem: { 'oneOf': [
			{ type: 'null' },
			{
				type: 'object',
				properties: {
					product_slug: { type: 'string' },
					free_trial: { type: 'boolean' }
				}
			}
		] },
		siteId: { type: 'string' },
		siteSlug: { type: 'string' }
	}
};
```